alpha/external-api-dropwizard/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:8-jdk
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ADD target/external-api-dropwizard-1.0-SNAPSHOT.jar /usr/src/app/target/external-api-dropwizard-1.0-SNAPSHOT.jar
```

Comment:
forget target folder on container, our build system folder is not relevant to container